### PR TITLE
 Rename base_service to mash_service.

### PR DESCRIPTION
**Pull request by smarlowucf**
And BaseService to MashService.

Update copyright notices.

### CODE REVIEW

File: mash/services/logger/service.py [hunk 1]
```diff
@@ -19,10 +19,10 @@
 import json
 
 from mash.mash_exceptions import MashLoggerException
-from mash.services.base_service import BaseService
+from mash.services.mash_service import MashService
 
 
-class LoggerService(BaseService):
+class LoggerService(MashService):
```
Comment (rjschwei) [inline mash/services/logger/service.py]: Shouldn't this be a PipelineService now? similar for others?

Comment (smarlowucf) [reply mash/services/logger/service.py]: No LoggerService isn't in the pipeline. It's more of an auxiliary service like Credentials and JobCreator. So it doesn't share all of the listener message methods that are in PipelineService.

File: mash/services/uploader/service.py [hunk 2]
```diff
@@ -32,7 +32,7 @@
 )
 
 
-class UploadImageService(BaseService):
+class UploadImageService(MashService):
```
Comment (rjschwei) [inline mash/services/uploader/service.py]: PipelineService?

**Comment by smarlowucf** [comment]
Rebased with master to fix merge conflicts.

**Pull request merged**